### ISSUE TITLE
[tsi-typescript] union types don't always indent properly

### ISSUE THREAD

**Issue by orzechowskid** [orzechowskid/tsi.el#2]
there's a unit test for that but it's currently failing, and is commented out.

expected behavior:
```typescript
type FooBar = 'foo'
  | 'bar';
```

actual behavior:
```typescript
type FooBar = 'foo'
| 'bar';
```

**Comment by orzechowskid** [comment]
CST:
```
program:
  type_alias_declaration:
    type_identifier:
    union_type:
      literal_type:
        string:
      literal_type:
        string:
```

**Comment by orzechowskid** [comment]
I think I fixed this in #5 :+1:
